src/plugins/tables/tables.js

```diff
@@ -185,7 +185,9 @@ $document.on( "submit", ".wb-tables-filter", function( event ) {
 
 	event.preventDefault();
 
-	var $form = $( this ),
+	var $options = $( "input[name='advanced_options']:checked" ).val(),
```

Comment:
````input[name='advanced_options']`` that seems very specific and not extendable or being configurable.

Aslo it will need to be documented

Reply:
As stated above these are only for checkboxes? Also they have to be user visible so using data attributes not really an option.

Reply:
Also if you have suggestions provide them.